INSTALL.md

```diff
@@ -383,35 +294,6 @@ Note that it is also discouraged, but you _can_ install PISA as an unprivileged
 This is not quite as clean as a virtual environment, and the issue with coflicting package dependency versions remains.
 
 
-### Install PISA
-
-```bash
-pip install -e $PISA[develop] -vvv
-```
-Explanation:
-* First, note that this is ***not run as administrator***. It is discouraged to do so (and has not been tested this way).
-* `-e $PISA` (or equivalently, `--editable $PISA`): Installs from source located at `$PISA` and  allows for changes to the source code within to be immediately propagated to your Python installation.
-Within the Python library tree, all files under `pisa` are links to your source code, so changes within your source are seen directly by the Python installation. Note that major changes to your source code (file names or directory structure changing) will require re-installation, though, for the links to be updated (see below for the command for re-installing).
-* `[develop]` Specify optional dependency groups. You can omit any or all of these if your system does not support them or if you do not need them.
-* `-vvv` Be maximally verbose during the install. You'll see lots of messages, including warnings that are irrelevant, but if your installation fails, it's easiest to debug if you use `-vvv`.
-* If a specific compiler is set by the `CC` environment variable (`export CC=<path>`), it will be used; otherwise, the `cc` command will be run on the system for compiling C-code.
-
-__Notes:__
-* You can work with your installation using the usual git commands (pull, push, etc.). However, these ***won't recompile*** any of the extension (i.e. pyx, _C/C++_) libraries. See below for how to reinstall PISA when you need these to recompile.
-
-
-### Reinstall PISA
-
-Sometimes a change within PISA requires re-installation (particularly if a compiled module changes, the below forces re-compilation).
-
-```bash
-pip install -e $PISA[develop] --force-reinstall -vvv
-```
-
-Note that if files change names or locations, though, the above can still not be enough.
-In this case, the old files have to be removed manually (along with any associated `.pyc` files, as Python will use these even if the `.py` files have been removed).
-
-
 ### Compile the documentation
 
 To compile a new version of the documentation to html (pdf and other formats are available by replacing `html` with `pdf`, etc.):
```

Comment:
Is it true that html can simply be replaced by pdf? There is no "pdf" target in the `Makefile` produced by Sphinx (8.1.3 in my case). The [Makefile](https://github.com/icecube/pisa/blob/master/docs/Makefile) we ship also doesn't contain this target.

Replace "(pdf and other formats are available by replacing `html` with `pdf`, etc.)" with "(run `make help` for other documentation formats).